src/stores/AccountDetailStore.ts

```diff
@@ -7,19 +7,23 @@ import AppStore from './AppStore';
 import Transaction from '../models/Transaction';
 
 export default class AccountDetailStore {
+  private static GET_TX_INTERVAL_MS: number = 60000;
+
   @observable public activeTabIdx: number = 0;
   @observable.shallow public items: Transaction[] = [];
   @observable public pageNum: number = 0;
   @observable public pagesTotal?: number;
 
-  constructor(private app: AppStore) {}
+  private getTransactionsInterval?: NodeJS.Timer = undefined;
 
   @computed public get hasMore(): boolean {
     return !!this.pagesTotal && (this.pagesTotal > this.pageNum + 1);
- }
+  }
+
+  constructor(private app: AppStore) {}
```

Comment:
dont think we need to set the visibility on a param? also, you need to define a class var for `app`

Reply:
as discussed, this is alternate syntax for class var declaration